client/assets/stylesheets/shared/functions/_z-index.scss

```diff
@@ -174,6 +174,7 @@ $z-layers: (
 		".is-section-woocommerce .global-notices": 999999,
 		".webpack-build-monitor": 99999999,
 		".people-list-item.card.is-compact:focus": 1,
+		".reader-join-conversation-dialog__overlay": 10000001,
```

Comment:
I had to increase the z-index of this join overlay, as the gravatar hovercards were higher in the index:

![Screenshot 2025-06-18 at 12 27 26 PM](https://github.com/user-attachments/assets/444a6d57-100f-4b6c-b5b5-2a0b5285edd6)